api/src/services/assets.ts

```diff
@@ -62,6 +62,9 @@ export class AssetsService {
 
 			const readStream = storage.disk(file.storage).getStream(file.filename_disk, range);
 			const transformer = sharp().rotate().resize(resizeOptions);
+			if (transformation.quality) {
+				transformer.toFormat(type.substring(6), { quality: Number(transformation.quality) });
+			}
```

Comment:
This would "just" transform it from input-type to output-type right? Just making sure we don't accidentally generate JPEGs from PNGs or something like that

Reply:
yes it will be based on the file type.